targets/ice40_hx8k_b_evn/base.py

```diff
@@ -80,15 +81,18 @@ def __init__(self, platform, **kwargs):
         # SoCCore ----------------------------------------------------------------------------------
         SoCCore.__init__(self, platform, sys_clk_freq, **kwargs)
 
-        self.cpu.use_external_variant("gateware/cpu/VexRiscv_Fomu_Debug.v")
+        if isinstance(self.cpu, VexRiscv):
+            self.cpu.use_external_variant("gateware/cpu/VexRiscv_Fomu_Debug.v")
```

Comment:
This does not compile on Travis at all.

@mithro - is this just a leftover of a debugging process, or should this file be included in this PR?